packages/frontend-shared/src/locales/en-US.json

```diff
@@ -504,6 +504,10 @@
     "experiments": {
       "title": "Experiments",
       "description": "If you'd like to try out new features that we're working on, you can enable beta features for your project by turning on the experimental features you'd like to try. {0}",
+      "experimentalCspAllowList": {
+        "name": "CSP Allow List",
+        "description": "Enables Cypress to selectively permit Content-Security-Policy header directives, including those that might otherwise block Cypress from running."
```

Comment:
do we need to mention `Content-Security-Policy-Report-Only` here?

Reply:
Good call. I'll add it.